azure-maven-plugin-lib/src/main/java/com/microsoft/azure/maven/AbstractAzureMojo.java

```diff
@@ -395,15 +395,15 @@ private static Account accountLogin(@Nonnull com.microsoft.azure.toolkit.lib.aut
                 }
                 // prompt if oauth or device code
                 promptForOAuthOrDeviceCodeLogin(account.getAuthType());
-                return handleDeviceCodeAccount(com.microsoft.azure.toolkit.lib.Azure.az(AzureAccount.class).login(account).account());
+                return handleDeviceCodeAccount(com.microsoft.azure.toolkit.lib.Azure.az(AzureAccount.class).loginAsync(account, false).block());
```

Comment:
1. no need to use qualified name
2. consider make `Azure.az(AzureAccount.class)` a field.
3. why not use `login(account, false)`?